### DISCUSSION TITLE
Control mapping for next/previous mapset

### DISCUSSION THREAD

**Discussion by toppasaurus** [ppy/osu#18078] [Ideas] [answered]
I recently set up all of my commonly used bindings on a MIDI pad controller, and noticed that mappings for next/previous mapset, normally the left and right arrow keys, weren't there. Not having that slows song navigation down slightly, as scrolling through one of those sets with 20+ diffs is a bit slow. I'd like to request that those bindings be added, please.

**Comment by frenzibyte** [comment, marked as answer]
Should be resolved in https://github.com/ppy/osu/pull/18082, but note that, with the PR as-is, you'll have to rebind the actions when switching between keyboard and MIDI pad controller, as you can only specify one key (or combination).
